### ISSUE TITLE
[A11y] Mobile Search Results Grid Checkboxes, Select All or None

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#2004]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Search Results-Grid: https://staging.oregondigital.org/catalog?&q%5B%5D=test&search_field=all_fields&utf8=%E2%9C%93&view=gallery

I am a non logged user. I am viewing the search results grid. Unlike the list view where the checkboxes are visible. They are not visible in grid view. As a user, I would need to know that the check box is hidden in the upper left corner to select the checkbox. Otherwise, if I click on the image it highlights it in black and also automatically checkbox the corner of the image.

When I click on select in grid view on mobile it does nothing, unlike list view where it does select all. However, with that said. I went back to double check this and I have different results.

So now, if I check list view and select all it does what its expected. However, when I swap it out for grid view. It now shows the blue icon box as selected but its not selecting the same images that was shown in list view.

Hence, there is something wonky between the list and grid view when you choose to select all versus when you manual check items.

### Can anybody QA or does this need an expert?

Needs expert

**Comment by straleyb** [comment]
Azle verified that this could be closed.
